apps/basic-scroll-snap/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```

Comment:
@mandy8055,
Update the title of the app.

Reply:
@ManujJiraUser 
Thanks for identifying the issue. Updated now.